### PR TITLE
chore(deps): bump all minor versions

### PR DESCRIPTION
**Pull request by balavishnuvj**
Updated minor reps by running `npm update`. Probably we should have bot to this 😅. I was looking into someother issue when I noticed this. Feel free to discard if this is not necessary. 

Can you give a follow up PR where deps have major changes.

### CODE REVIEW

**Comment by smeijer** [comment]
Thanks Bala! I'll check and merge soon. Regarding the bot, we had it. But it's kind of annoying. It keeps us busy with updating, even when things aren't broken. I'd rather do bulk updates once in a while.

**Comment by marcosvega91** [comment]
A bot could be more a pain than a help. Sometime is better to update every month everything. 

Anyway we don't have tests for all the codebase so updating libraries could be a problem xD.

File: package.json [hunk 1]
```diff
@@ -39,70 +39,70 @@
...
-    "@primer/octicons-react": "^10.0.0",
+    "@primer/octicons-react": "^10.1.0",
     "@reach/dialog": "^0.10.5",
     "@reach/menu-button": "^0.9.1",
     "@testing-library/dom": "^7.28.1",
-    "@testing-library/user-event": "^12.1.2",
+    "@testing-library/user-event": "^12.5.0",
     "codemirror": "5.54.0",
-    "crx-bridge": "^2.1.0",
+    "crx-bridge": "^3.0.1",
```
Comment (smeijer) [inline package.json:48]: I'm not sure if it's a good thing to add this major bump here, as crx-bridge has changed quite a bit to add support for Firefox.

Should this one be reverted?

Comment (balavishnuvj) [reply package.json:48]: I had tested it worked. But I didn't any release notes. So should I revert it?

Comment (smeijer) [reply package.json:48]: Did you test the build & functioning of the chrome extension? That's where `crx-bridge` is being used.

If it works, we can merge it.

**Comment by smeijer** [comment]
@balavishnuvj, can you fix the conflicts? :innocent: I'll hit merge right after.

**Comment by balavishnuvj** [comment]
will fix them today!

**Comment by balavishnuvj** [comment]
@smeijer have updated. Also have reverted `"crx-bridge"` to v2 for now.

**Comment by smeijer** [comment]
Thanks, Bala!

**Comment by smeijer** [comment]
@all-contributors please add @balavishnuvj for code

**Pull request merged**